org.scala-ide.sdt.core/src/org/scalaide/ui/internal/editor/ScalaContentProvider.scala

```diff
@@ -0,0 +1,86 @@
+package org.scalaide.ui.internal.editor
+import org.eclipse.jface.viewers.ITreeContentProvider
+import org.eclipse.jface.viewers.Viewer
+
+
+
+
+/**
+ * @author ailinykh
+ */
```

Comment:
Please don't put `@author` tags anywhere. There is an `AUTHORS` file in the root dir.